config/dev.config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
don't you prefer `config.dev.js`, that sound like `webpack.dev.js`

Reply:
changed